armbian/base/scripts/systemd-start-bitcoind.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
```

Comment:
(Optional) Could we add a brief comment explaining what these scripts do? Maybe something like:

```
#!/bin/bash
#
# This script is called by the bitcoind.service to start bitcoind.
#
```

(And elsewhere.)